packages/backend/src/index.ts

```diff
@@ -140,7 +140,7 @@ async function main() {
 
   await service.start().catch(err => {
     logger.error(err);
-    process.exit(1);
+    throw err;
```

Comment:
Yeah I put this change in here but didn't make the corresponding thing in create-app - this is all not working as intended anyway, so pending a rewrite of the backend lifecycle, I'll keep the template as is

Reply:
So picking up the discussion, should we just keep this one as is?

On the topic of how process exit acts, I'm assuming this is what we were talking about:

```bash
node -e "require('http').createServer().listen(() => {})"
# vs
node -e "require('http').createServer().listen(() => {process.exit(1)})"
```

Reply:
More like

```ts
node -e "setInterval(() => {}, 1000); require('http').createServer().listen(() => {process.exit(1)})"
```

Wherein there's still an active timer at the time of the demise of the process. But yeah I just tried that and it does indeed work. So the remaining thing then, is probably that the webpack dev server just keeps it alive - for better or worse - and maybe we can handle that somehow.

Reply:
Reverted this one